enum.dd

```diff
@@ -242,7 +242,7 @@ enum
 $(H2 Manifest Constants)
 
 	$(P If there is only one member of an anonymous enum, the { } can
-	be omitted:
+	be omitted. Gramatically speaking, this is an $(GLINK2 declaration, AutoDeclaration).
```

Comment:
"Gramatically speaking" is a bit casual. "Grammatically" should be enough. I'll let this be though.